edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/RouteConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System.ComponentModel;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Routing.Core;
+    using Newtonsoft.Json;
+
+    public class RouteConfiguration

```

Comment:
Would it be possible to have only 1 implementation, either RouteConfig and RouteConfiguration? If really not possible, please think about to rename RouteConfiguration class in order to easy distinguish from existing RouteConfig class.

Reply:
yea, I noticed this as well.  I plan on renaming this to RouteSpec after I get a sign-off.  I figured if I rename this during the PR, then the commenting will get really really weird.

---
In reply to: [377442628](https://github.com/Azure/iotedge/pull/2469#discussion_r377442628) [](ancestors = 377442628)

Reply:
or rename in a separate PR to reduce confusion.

---
In reply to: [377814817](https://github.com/Azure/iotedge/pull/2469#discussion_r377814817) [](ancestors = 377814817,377442628)